internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -54,6 +55,9 @@ ServiceConnectConfiguration:
           {{- else}}
           DnsName: !Ref WorkloadName
           {{- end}}
+  {{- else}}
+  Enabled: False
```

Comment:
What was the issue with the original snippet? When sc is not enabled, how does CFN/ECS handle it differently between removing the entire `ServiceConnectConfiguration`  vs. setting `Enabled: False`?

Reply:
If we don't set `ServiceConnectConfiguration` CFN keeps using the old template 🤦 If you want to disable this feature you have to explicitly specify in the template.